Ark.Tools.AspNetCore.ApplicationInsights/Startup/Ex.cs

```diff
@@ -18,11 +18,11 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Ark.Tools.AspNetCore.ApplicationInsights
+namespace Ark.Tools.AspNetCore.ApplicationInsights.Startup
 {
-    public static class HostedServiceConfiguration
+    public static partial class Ex
     {
-        public static void ConfigureServices(IConfiguration configuration, IServiceCollection services)
+        public static IServiceCollection ConfigureServicesWebHostArk(this IServiceCollection services, IConfiguration configuration)
```

Comment:
this isn't 'services' in a generic way, this is for ApplicationInsightsTelemetry only